### PR TITLE
495 change geoips get to geoips describe

### PR DESCRIPTION
**Pull request by evrose54**
# Reviewer Checklist

* [x] Required ***existing tests*** pass (ie full_test.sh, others as appropriate)
* [x] Required ***unit tests***  pass and modified for refactored functionality
* [x] NO REQUIRED ***integration tests*** (explain why not required)
* [x] Required ***documentation*** added for new/modified functionality
* [x] Required ***release notes*** added for new/modified functionality
* [x] NO REQUIRED ***updates to other repos*** (explain why not required)

# Related Issues
fixes NRLMMD-GEOIPS/geoips#495

# Testing Instructions
Run ``pytest -v /tests/unit_tests/commandline/`` and get all passing tests.

# Summary
This update renames all references of 'geoips get' to 'geoips describe'. 'Describe'
is a better name for this command as it generally provides additional information
about a certain GeoIPS artifact, whereas 'get' could be a bit ambiguous as users
may think we are actually getting a certain plugin, interface, etc. Another reason
for this update is that all references to the CLI in the tutorial use
'geoips describe', rather than 'geoips get'. This PR was only a nomenclature change
and did not impact any of the actual functionality of the CLI.

### CODE REVIEW

File: geoips/commandline/ancillary_info/cmd_instructions.yaml [hunk 1]
```diff
@@ -17,15 +17,15 @@ instructions:
       To use, type `geoips config install <test_dataset_name>`.
     output_info:
       - Not Applicable
-  geoips_get: &geoips-get-interface
+  geoips_describe: &geoips-describe-artifact
     help_str: |
       Retrieve information about a GeoIPS artifact. For this command, an artifact is one
-      of ['interface', 'plugin', 'family']. To get more information on how to retrieve
-      these artifacts, see below.
+      of ['interface', 'plugin', 'family']. To describe more information on how to
```
Comment (jsolbrig) [inline geoips/commandline/ancillary_info/cmd_instructions.yaml:23]: "describe" seems inappropriate here.

Comment (evrose54) [reply geoips/commandline/ancillary_info/cmd_instructions.yaml:23]: Ah, that was a mistake on my part. I was just doing a find and replace and accidentally included that 'get'. I'll change that back. Let me know if you find any more of those!

File: geoips/commandline/ancillary_info/cmd_instructions.yaml [hunk 2]
```diff
@@ -214,8 +214,8 @@ instructions:
       List off every GeoIPS Interface. This is useful for seeing what is
       available throughout GeoIPS, but can also be extended to see what interfaces have
       been implemented in other geoips package, or just a certain package. This
-      information can then be used with `geoips get-interface <interface_name>` to get
-      more information about a certain interface.
+      information can then be used with `geoips describe-interface <interface_name>` to
```
Comment (jsolbrig) [inline geoips/commandline/ancillary_info/cmd_instructions.yaml:217]: Should "describe-interface" be "describe interface"?

File: geoips/commandline/ancillary_info/cmd_instructions.yaml [hunk 3]
```diff
@@ -233,8 +233,8 @@ instructions:
   geoips_list_plugins:
     help_str: |
       List off plugins found under all, or a specified GeoIPS package. This information
-      can then be used with `geoips get-plugin <interface_name> <plugin_name>` to get
-      more information about a certain plugin.
+      can then be used with `geoips describe-plugin <interface_name> <plugin_name>` to
```
Comment (jsolbrig) [inline geoips/commandline/ancillary_info/cmd_instructions.yaml:236]: Should "describe-plugin" be "describe plugin"?

File: geoips/commandline/ancillary_info/cmd_instructions.yaml [hunk 4]
```diff
@@ -233,8 +233,8 @@ instructions:
...
+      describe more information about a certain plugin.
```
Comment (jsolbrig) [inline geoips/commandline/ancillary_info/cmd_instructions.yaml:237]: Is "describe" appropriate here?

File: geoips/commandline/ancillary_info/cmd_instructions.yaml (hunk 1 above)
Comment (jsolbrig) [reply geoips/commandline/ancillary_info/cmd_instructions.yaml:23]: I hope @mindyls has told you about "single_varnel".

File: geoips/commandline/ancillary_info/cmd_instructions.yaml [hunk 5]
```diff
@@ -176,8 +176,8 @@ instructions:
       List off plugins found under this interface. This is useful for seeing what
       plugins have been implemented through all, or a certain, GeoIPS Package. This
       information can then be used with
-      `geoips get <interface_name> <plugin_name>` to get more information about a
-      certain plugin.
+      `geoips describe <interface_name> <plugin_name>` to describe more information
```
Comment (jsolbrig) [inline geoips/commandline/ancillary_info/cmd_instructions.yaml]: Again here.

File: geoips/commandline/ancillary_info/cmd_instructions.yaml [hunk 6]
```diff
@@ -214,8 +214,8 @@ instructions:
       List off every GeoIPS Interface. This is useful for seeing what is
       available throughout GeoIPS, but can also be extended to see what interfaces have
       been implemented in other geoips package, or just a certain package. This
-      information can then be used with `geoips get-interface <interface_name>` to get
-      more information about a certain interface.
+      information can then be used with `geoips describe-interface <interface_name>` to
+      describe more information about a certain interface.
```
Comment (jsolbrig) [inline geoips/commandline/ancillary_info/cmd_instructions.yaml]: Again here

File: geoips/commandline/ancillary_info/cmd_instructions.yaml (hunk 2 above)
Comment (evrose54) [reply geoips/commandline/ancillary_info/cmd_instructions.yaml]: Fixed! Should just be ``geoips describe <interface_name>``. This was outdated apparently.

File: geoips/commandline/ancillary_info/cmd_instructions.yaml (hunk 3 above)
Comment (evrose54) [reply geoips/commandline/ancillary_info/cmd_instructions.yaml]: Fixed! Should just be ``geoips describe <interface_name> <plugin_name>``. This was outdated apparently.

File: geoips/commandline/ancillary_info/cmd_instructions.yaml (hunk 4 above)
Comment (evrose54) [reply geoips/commandline/ancillary_info/cmd_instructions.yaml]: Fixed this as well. Find and replace mistakes here.

File: geoips/commandline/ancillary_info/cmd_instructions.yaml (hunk 5 above)
Comment (evrose54) [reply geoips/commandline/ancillary_info/cmd_instructions.yaml]: Fixed.

File: geoips/commandline/ancillary_info/cmd_instructions.yaml (hunk 6 above)
Comment (evrose54) [reply geoips/commandline/ancillary_info/cmd_instructions.yaml]: Fixed.

File: geoips/commandline/geoips_describe.py [hunk 7]
```diff
@@ -19,17 +19,17 @@
 from geoips import interfaces
 
 
-class GeoipsGetInterface(GeoipsExecutableCommand):
-    """Get Command which retrieves information about a from or about a GeoIPS interface.
+class GeoipsDescribeArtifact(GeoipsExecutableCommand):
+    """Command which describes information about a GeoIPS artifact.
```
Comment (jsolbrig) [inline geoips/commandline/geoips_describe.py:23]: Maybe "Command which returns information describing a GeoIPS artifact."?

Comment (evrose54) [reply geoips/commandline/geoips_describe.py:23]: Sure, I'll make that change. I thought that sounded weird too.

Comment (evrose54) [reply geoips/commandline/geoips_describe.py]: Fixed.

**Review by jsolbrig — APPROVED** [review]

File: geoips/commandline/geoips_command.py [hunk 8]
```diff
@@ -573,12 +573,12 @@ class CommandClassFactory:
...
-    * GeoipsGetInterface
+    * GeoipsDescribeArtifact
 
-        * GeoipsGetInterfaceAlgorithm
-        * GeoipsGetInterfaceColormapper
+        * GeoipsGetArtifactAlgorithm
+        * GeoipsGetArtifactColormapper
         * ...
-        * GeoipsGetInterfaceTitleFormatter
+        * GeoipsGetArtifactTitleFormatter
```
Comment (mindyls) [inline geoips/commandline/geoips_command.py:581]: Should these also be GeoipsDescribeArtifact?  Not GeoipsGet?

**Review by mindyls — APPROVED** [review]
Just left one comment, looks good!

File: geoips/commandline/geoips_command.py (hunk 8 above)
Comment (evrose54) [reply geoips/commandline/geoips_command.py]: Yes; Good catch! Fixed this now.

**Review by mindyls — APPROVED** [review]
Looks great, thanks!  Excited to get everything cleaned up for the workshop!

**Review by jsolbrig — APPROVED** [review]

**Pull request merged**